### PR TITLE
LLM: don't export env variables when linux kernel is 6.5

### PR DESCRIPTION
**Pull request by WeiguangHan**
@hkvision Don't export env variables when linux kernel is 6.5

### CODE REVIEW

**Review by hkvision — APPROVED** [review]

**Comment by hkvision** [comment]
Also need to update run-max-gpu.sh?

**Comment by WeiguangHan** [comment]
> Also need to update run-max-gpu.sh?

sure

File: python/llm/dev/benchmark/all-in-one/run-arc.sh [hunk 1]
```diff
@@ -1,5 +1,9 @@
 source /opt/intel/oneapi/setvars.sh
+
 export USE_XETLA=OFF
-export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
+KERNEL_VERSION=$(uname -r)
+if [[ $KERNEL_VERSION != *"6.5"* ]]; then
```
Comment (qiuxin2012) [inline python/llm/dev/benchmark/all-in-one/run-arc.sh:5]: what if 6.5 and higher?

Comment (hkvision) [reply python/llm/dev/benchmark/all-in-one/run-arc.sh:5]: we don't set `SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS` for 6.5 or higher versions, namely do nothing.

**Pull request closed without merging**